magda-registry-api/src/test/scala/au/csiro/data61/magda/registry/WebHookProcessingSpec.scala

```diff
@@ -81,27 +87,39 @@ class WebHookProcessorSpec extends ApiSpec {
         status shouldEqual StatusCodes.OK
       }
 
-      val result2 = Await.result(processor.sendSomeNotificationsForOneWebHook("test"), 5 seconds)
-      result2.statusCode should be(Some(StatusCodes.OK))
+      Util.waitUntilDone(actor, "test")
 
       payloads.length shouldBe 1
       payloads(0).events.get.length shouldBe 1
-      payloads(0).records.get.length shouldBe 0
+      payloads(0).records.get.length shouldBe 1
```

Comment:
I'm not sure this test really makes sense anymore.  Or its description  is wrong or something.  Previously it did this:
1. Set up a webhook that only cares about aspect A.
2. Modify a record's aspect B.
3. Assert that we get an event but _not_ a record in the webhook payload.

In other words, it checks that the payload `includes records only for events modifying aspects that were requested`.

But with the changes in this PR, events are filtered by aspects, too.  So we won't get any events, and so we won't get any payload at all.  It could be useful to have a test for that, but that's not what this test is anymore, because the webhook is now subscribed to aspect B, too.

So maybe just delete the word "only" from this spec's description, i.e. "includes records for events modifying aspects that were requested."  Adding that "no payload generated" test mentioned above would be great, too, if it's not hard.